src/main/java/school/lemon/changerequest/java/introduction/hw1/Task9.java

```diff
@@ -2,7 +2,11 @@
 
 public class Task9 {
 
-    public static boolean isPowerOfThree(int n) {
-        return false;
+    public static boolean isPowerOfThree(int n)
+    {
+        double Rezult=Math.log(n)*(int)(Math.log(3))/((int)(Math.log(n))*(Math.log(3)));
```

Comment:
It's pretty cool that you've used Math.log, but could you please do it with simple loop using `/` and `%` operations. It might be useful.